ui/tests/e2e/test.js

```diff
@@ -39,6 +39,17 @@ describe("End-to-end", () => {
     expect(grayDiv).toBeTruthy();
   });
 
+  test("Export to Saturn With Facet", async () => {
```

Comment:
Let's do [this comment](https://github.com/DataBiosphere/data-explorer/pull/139#discussion_r214208134) so all tests start from a clean slate.